packages/jspsych/src/JsPsych.ts

```diff
@@ -364,6 +364,25 @@ export class JsPsych {
     return this.file_protocol;
   }
 
+  getTimelineDescription() {
+    var trials = [];
+    function getTrialsFromTimelineNodes(timeline_node) {
+      if (typeof timeline_node.timeline_parameters !== "undefined") {
+        for (var i = 0; i < timeline_node.timeline_parameters.timeline.length; i++) {
+          if (
+            typeof timeline_node.timeline_parameters.timeline[i].trial_parameters !== "undefined"
+          ) {
+            trials.push(timeline_node.timeline_parameters.timeline[i].trial_parameters);
+          } else {
+            getTrialsFromTimelineNodes(timeline_node.timeline_parameters.timeline[i]);
+          }
+        }
+      }
+    }
+    getTrialsFromTimelineNodes(this.timeline);
+    return trials;
+  }
```

Comment:
Very slick! You will probably hate me for this, but what about
```suggestion
  getTimeline() {
    return this.timelineDescription
  }
```
adding
```ts
private timelineDescription;
```
to the private variables section, and setting
```ts
this.timelineDescription = timeline;
```
in `run()`?

Reply:
Ha! Yep that's much better 😄

Reply:
Done in [`79b0665`](https://github.com/jspsych/jsPsych/commit/79b06658ec5a27d563d64b1338b3be7befb4efe6) 👍